bgpd/bgpd.h

```diff
@@ -2909,7 +2909,7 @@ static inline bool peer_dynamic_neighbor_no_nsf(struct peer *peer)
 
 static inline int peer_cap_enhe(struct peer *peer, afi_t afi, safi_t safi)
 {
-	return (CHECK_FLAG(peer->af_cap[afi][safi], PEER_CAP_ENHE_AF_NEGO));
+	return (peer && CHECK_FLAG(peer->af_cap[afi][safi], PEER_CAP_ENHE_AF_NEGO));
```

Comment:
I think I would prefer a assert(peer);  There is no reasonable call path that we ever get here where peer is non-null.  I'd like to find that call path and fix it instead of silently moving on.  I agree that this is a theoretical possibility given the code but the `if (path)` code is followed every time on 719 in bgp_updgrp_packet.c